tiledb/sm/filesystem/posix.cc

```diff
@@ -455,23 +437,18 @@ Status Posix::read(
         std::string("Cannot read from file ' ") + path.c_str() +
         "'; offset > typemax(off_t)"));
   }
-  if (nbytes > SSIZE_MAX) {
+  if (nbytes > SIZE_MAX) {
```

Comment:
Isn't this check (now, with unsigned SIZE_MAX) useless, physical impossibility with both unsigned values?
[Maybe not in 32bit build, are we still supporting somewhere?]